src/main/java/org/jenkinsci/plugins/parallel_test_executor/SplitStep.java

```diff
@@ -42,14 +42,14 @@ public DescriptorImpl() {
 
     }
 
-    public static final class Execution extends AbstractSynchronousStepExecution<List<List<String>>> {
+    public static final class Execution extends AbstractSynchronousStepExecution<List<InclusionExclusionPattern>> {
 
         @Inject private SplitStep step;
         @StepContextParameter private Run<?,?> build;
         @StepContextParameter private TaskListener listener;
 
-        @Override protected List<List<String>> run() throws Exception {
-            return ParallelTestExecutor.findTestSplits(step.parallelism, build, listener);
+        @Override protected List<InclusionExclusionPattern> run() throws Exception {
```

Comment:
This is an incompatible change I think.

[example](https://github.com/jenkinsci/parallel-test-executor-plugin/blob/7bdb270ffe02838a36f56d4de16ebe61cd3a9556/demo/JENKINS_HOME/jobs/flow/config.xml#L12-L15)